lib/template/pages/schedule.jade

```diff
@@ -150,7 +150,7 @@
               =' '
               | Vstupenky
             =' '
-            | První seznámení s improvizací
+            | První seznámení s improvizací, ImproInstitut
```

Comment:
Ajo, dik. Změnil jsem.

Pája

On 13 June 2016 at 16:25, Pavel Žák <notifications@github.com> wrote:

> In lib/template/pages/schedule.jade
> <https://github.com/just-paja/zpatra-web/pull/15#discussion_r66800421>:
>
> > @@ -150,7 +150,7 @@
> >                =' '
> >                | Vstupenky
> >              =' '
> > -            | První seznámení s improvizací
> > +            | První seznámení s improvizací, ImproInstitut
>
> Nepíšou se s mezerou?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/just-paja/zpatra-web/pull/15/files/1f3627b9b1e2852c75c5919f18b9d39cc9ba2a36#r66800421>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASX3oIAoBFPJzG0qb_dqX3rSe0F1LbAqks5qLWhWgaJpZM4I0Qrd>
> .
>